### PR TITLE
TRUNK-4976 - Support obs.interpretation and obs.status, aligned with FHIR

### PR DESCRIPTION
**Pull request by djazayeri**
I will merge this myself. -Darius

### CODE REVIEW

**Comment by wluyima** [comment]
Is client code the one expected to change the status to amended when an Obs is edited?

**Comment by djazayeri** [comment]
@wluyima, Burke and I discussed this on the design call yesterday, and we think it would be better if the API sets this automatically. I'm viewing that as a nice-to-have for now, and as it is now the code doesn't do this, and the client is expected to set this field.

**Comment by wluyima** [comment]
Okay, thanks!

File: api/src/main/java/org/openmrs/Obs.java [hunk 1]
```diff
@@ -64,6 +64,32 @@
  */
 public class Obs extends BaseOpenmrsData {
 	
+	public enum Interpretation {
```
Comment (wluyima) [inline api/src/main/java/org/openmrs/Obs.java]: Wouldn't it be nice to have @since for these new enums?

Comment (djazayeri) [inline api/src/main/java/org/openmrs/Obs.java]: Good point, I remembered for the getters/setters but I forgot it here. I will add this.

**Pull request merged**